controllers/logout.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res) => {
+  req.logout();
```

Comment:
hmm so is this provided by `cookie-session` ?

Reply:
Yeah, it is provided. And it works